Fix to build on FreeBSD.

mtravis15432 opened the pull request:
These changes allow rtags to build on FreeBSD 10. I'm not certain how to properly implement FileSystemWatcher::shutdown() so I left the function definition empty.

Andersbakken commented:
Ah. I mixed up shutdown and cleanup when I fixed this (don't have a FreeBSD machine to test on). I've fixed it now. Thanks for the merge request.

mtravis15432 commented:
Thank you for fixing it.

On Fri, Nov 27, 2015 at 1:55 PM, Anders Bakken <notifications@github.com>
wrote:

> Ah. I mixed up shutdown and cleanup when I fixed this (don't have a
> FreeBSD machine to test on). I've fixed it now. Thanks for the merge
> request.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Andersbakken/rct/pull/44#issuecomment-160217885>.
>



-- 
Mark Travis
Founder, InfiniSQL
mtravis@infinisql.org
http://www.infnisql.org
"Extreme Scale Transaction Processing"
